projects/flow.hoomd.lj/README.md

```diff
@@ -1,53 +1,66 @@
+    - You can tell which methods are operations because they will have the `@MyProject.operation` decorator, which tells signac that this method is to be treated as an operation.
+    - `operation`s typically have pre- and post- conditions. This is how signac knows when the operation should be run. For example, the `initialize()` function has no pre-condition because it must be run first, but has `@MyProject.post.isfile('init.gsd')` as a post-condition. This post-condition means that the operation will *not* be run if `init.gsd` exists.
+
+3. Now run let's run the operations:
+
+- First run a status check:
+
+    ```
+    python src/project.py status -d
+    ```
 
-We start by initializing the project and a few state points with the `init.py` module.
+    `-d` specifies a "detailed view". You will see a list of jobs and that `estimate` and `initialize` are eligible operations, as determined by those operations' pre- and post- conditions.
 
-```
-python src/init.py 42
-```
+- Now initialize eligible jobs:
 
-The number 42 is the random seed used for initialization, feel free to replace it with a different number or a text string, which will be converted into a numeric random seed.
+    ```
+    python src/project.py run -o initialize
+    ```
 
-The project's workflow and data space operations are defined within the `src/project.py` module.
-We can check the project's status, e.g., with:
+    This will run just the `initialize()` operations for *all* eligible jobs (which in this case is all of the jobs).
+    - Run `python project.py status -d` again, and you'll see that now all the jobs are eligible for `estimate` and `sample`. You can also run `python project.py status -d -p p`. The ``-p`` argument specifies which parameters should be shown in the status view, and we pass in `p` to see which statepoint corresponds to which pressure.
 
-```
-python src/project.py status --detailed --parameters p
-```
+- Now we can run the simulations:
 
-We use the `--detailed` flag to show the labels explicitly for each job.
-The `--parameters` (`-p`) argument specifies state point parameters that should be shown in the status overview.
-In this case we specify to show the `p` variable, which stands for pressure.
+    ```
+    python src/project.py run
+    ```
 
-The status will also show all pending operations (initialize/estimate-volume/sample).
-The command
+    which will now run all eligible operations, and you'll see HOOMD be called. When you call `python project.py status -d` now, you'll see that no operations are eligible, and that the labels `estimated`, `sampled`, and `started` are now visible. These labels are defined in `project.py` with the `@MyProject.label` decorator.
 
-```
-python src/project.py run
-```
+4. For more information on how you can analyze this data, imply execute ``jupyter notebook`` within the project's root directory and open the `src/analysis.ipynb` notebook.
```

Comment:
This was supposed to say "simply" instead of "imply" but I think removing the word is better.
```suggestion
4. For more examples of how you can analyze this data, execute ``jupyter notebook`` within the project's root directory and open the `src/analysis.ipynb` notebook.
```